### PR TITLE
feature/implement linter suggestions

### PR DESCRIPTION
**Pull request by mr-nick17**
### What

Ignore linter warnings generated for console use 

### How to review

Sense check

### Who can review

!me

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -8,5 +8,7 @@
         "ecmaVersion": "latest",
         "sourceType": "module"
     },
-    "rules": {}
+    "rules": {
+        "no-console": "off"
```
Comment (lindenmckenzie) [inline .eslintrc.json:12]:
```suggestion
        "no-console": ["error", { allow: ["warn", "error"] }]
```

Comment (lindenmckenzie) [reply .eslintrc.json:12]: if that's what you're looking for anyway. Certainly don't think we should be allowing `console.log`

**Review by lindenmckenzie — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (mr-nick17) [reply .eslintrc.json:12]: hmm it gets messy implementing this globally. Inspiration taken from [eslint docs](https://eslint.org/docs/latest/rules/no-console#when-not-to-use-it). The config would end up looking like 
```json
 "rules": {
        "no-console": "off",
        "no-restricted-syntax": [
            "warn",
            {
                "selector": "CallExpression[callee.object.name='console'][callee.property.name!=/^(warn|error)$/]",
                "message": "Unexpected console statement"
            }
        ]
    }
```

Comment (lindenmckenzie) [reply .eslintrc.json:12]: that's only for if you want to allow console overwrites but not console calls no? 

my snippet does work for erroring on console.log but not console.error

Comment (mr-nick17) [reply .eslintrc.json:12]: ok so your snippet is not valid json hence why I went down the docs route. This snippet works...
```json
 "no-console": [
            "error",
            {
                "allow": [
                    "warn",
                    "error"
                ]
            }
        ]
```

Comment (lindenmckenzie) [reply .eslintrc.json]: you miss two quote marks and the world goes mad :(

**Review by lindenmckenzie — APPROVED** [review]

**Pull request merged**